client/src/main/java/se/fortnox/reactivewizard/client/ReactorRxClientProvider.java

```diff
@@ -58,8 +62,13 @@ private HttpClient buildClient(InetSocketAddress socketAddress) {
                 })
             )
             .port(config.getPort())
-            .doOnRequest((httpClientRequest, connection) -> healthRecorder.logStatus(connectionProvider, true))
-            .doOnError((httpClientRequest, throwable) -> healthRecorder.logStatus(connectionProvider, false), (httpClientResponse, throwable) -> { })
+            .doOnRequest((httpClientRequest, connection) -> {
```

Comment:
With doOnRequest we reset the errorcounter before the request is sent, just after it has been prepared. Shouldn't we reset the counter after the response has been received fully but before handing it off to the handler?

Reply:
Yes, but the connection is established so I think it is fine.